app/js/conceptSearch.controller.js

```diff
@@ -1,6 +1,14 @@
-angular
-    .module('conceptDictionaryApp')
-    .controller('ConceptSearchController', ['$scope', '$routeParams' ,'openmrsRest', function($scope, $routeParams, openmrsRest) {
+(function(){
+	'use strict';
+	
+	angular
+   		.module('conceptDictionaryApp')
+   		.controller('ConceptSearchController', ConceptSearchController)
+   	
+   	ConceptSearchController.$inject = ['$scope', '$routeParams' ,'openmrsRest', '$timeout']
+   				
+	
+	function ConceptSearchController($scope, $routeParams, openmrsRest, $timeout) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function has too many statements. (44)](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2330693977/issues/source?bid=3196969&fileBranchId=3207601#l11)